packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -0,0 +1,93 @@
+
+Note that your application needs to verify user identity through a user store, otherwise your app could be spoofed by fake applications in the Android or Apple app stores or an end user could view and possibly modify your code. For this reason, native, mobile, and single page application (SPA) integrations are not acceptable for OIDC app integrations in the OIN. Your application code must be set up in a flow so that your client application talks to your SaaS back-end which then communicates with Okta.
+
+To support the potentially large numbers of Okta orgs accessing it through the OIN, an OIDC integration can't use a custom authorization server, including the `default` server.
+
+A full guide to setting up the application to use the authorization code flow can be found in our Okta developer guide: [Implement the Authorization Code Flow](/docs/guides/implement-auth-code/overview/)
+
+Also, if you’ve used SAML for SSO in the past, it’s important to realize that the OIDC flow is different. The OIDC protocol doesn't just provide an assertion that is exchanged between Okta and your SaaS back-end, but uses a long-term token that can be used for callback into Okta at any point as long as the token is valid.
+
+### Scopes
+
+Your OIDC client needs to use scope values to define the access privileges that are being requested for access tokens. The scopes associated with access tokens determine what resources are available when the tokens are used to access the protected endpoints. Scopes can be used to request that specific sets of values be available as claim information about the end user.
+
+The only scope that must be declared is `openid`. When the authentication request is sent to Okta, the `openid` scope identifies the request as being an OIDC request.
+
+Other optional scopes available (these are returned from the `/userinfo` endpoint):
+
+* `profile` &mdash; the end user's default profile claims: `name`, `family_name`, `given_name`, `middle_name`, `nickname`, `preferred_username`, `profile`, `picture`, `website`, `gender`, `birthdate`, `zoneinfo`, `locale`, and `updated_at`
+* `email` &mdash; requests access to the `email` and `email_verified` claims
+* `address` &mdash; requests access to the `address` claim
+* `phone` &mdash; requests access to the `phone_number` and `phone_number_verified` claims
+* `groups` &mdash; requests access to the `groups` claim. This is a custom scope for Okta
+* `offline_access` &mdash; requests that a refresh token be generated that can be exchanged for an access token that can then be used to access the end user’s `/userinfo` endpoint even if the end user is not signed on
+
+As an OIDC app integration destined for the OIN can't use a custom authorization server, including the `default` server, the only customization you can make to the default scopes is to configure a custom `groups` claim.
```

Comment:
this statement is very confusing to the reader. Atleast I am confused.

Reply:
Checking on this ...

Reply:
These are somewhat separate concerns and probably deserve their own callouts- OIN clients can _only_ use the Org Authorization server, AND OIN clients can _only_ use OIDC scopes (yes, that excludes the `groups` scope today)

Reply:
Looking this over, I found that I already covered the Org authorization server restriction on line 42 of this page.
This means that the text here could just be shortened to cover the OIDC scope restriction:
> You can only request the [OIDC scopes](/docs/reference/api/oidc/#scopes). Custom scopes like `groups` can't be configured.